libnopegl/src/api.c

```diff
@@ -141,8 +141,8 @@ static const char *get_cap_string_id(unsigned cap_id)
 
 static int load_caps(struct ngl_backend *backend, const struct gpu_ctx *gpu_ctx)
 {
-    const int has_compute        = NGLI_HAS_ALL_FLAGS(gpu_ctx->features, NGLI_FEATURE_COMPUTE);
-    const int has_ds_resolve     = NGLI_HAS_ALL_FLAGS(gpu_ctx->features, NGLI_FEATURE_DEPTH_STENCIL_RESOLVE);
+    const uint32_t has_compute        = NGLI_HAS_ALL_FLAGS(gpu_ctx->features, NGLI_FEATURE_COMPUTE);
+    const uint32_t has_ds_resolve     = NGLI_HAS_ALL_FLAGS(gpu_ctx->features, NGLI_FEATURE_DEPTH_STENCIL_RESOLVE);
```

Comment:
These are boolean so `int` is appropriate; I'd rather cast explicitly below when fit into their u32 container

Reply:
Commit dropped.